tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -20,10 +20,12 @@
 #include "dxc/DxilContainer/DxilContainerAssembler.h"
 #include "dxc/DxilContainer/DxilPipelineStateValidation.h"
 #include "dxc/DxilHash/DxilHash.h"
+#include "dxc/Support/Unicode.h" // for wstring conversions like WideToUtf8String
 #include "dxc/Support/WinIncludes.h"
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/StringRef.h"
 #include "llvm/Support/Regex.h"
+#include <wchar.h> // for _wgetenv
```

Comment:
not 100%  sure why Damyan's comment here is resolved but I agree with it. Commenting on usage is a bit strange